File: Daily challenge/Make The String Great.js
/**
 * @param {string} s
 * @return {string}
 */
var makeGood = function(s) 
{
    let res = "";
    for (let i = 0; i < s.length; i++) 
    {
        if (res && Math.abs(s.charCodeAt(i) - res.charCodeAt(res.length - 1)) === 32) 
        {
            res = res.slice(0, -1);
        } 
        else 
        {
            res += s[i];
        }
    }
    return res;
};

File: Daily challenge/Make The String Great.py

class Solution(object):
    def makeGood(self, s):
        res = ""
        for letter in s:
            if res != "" and abs(ord(letter) - ord(res[-1])) == 32:
                res = res[:-1]
            else:
                res += letter
        
        return res
        